config.toml

```diff
@@ -39,7 +39,7 @@ enable = true
 [delegate]
 enable = true
 enable_deploy_identity = true
-fees = []
+fees = [{'base_fee'= 0, 'currency_network'= '0x0000000000000000000000000000000000000000', 'gas_price'= 0}]
```

Comment:
This is not a good default as it makes no sense, please remove, I left it out on purpose